llvm/include/llvm/IR/Intrinsics.h

```diff
@@ -173,9 +173,9 @@ namespace Intrinsic {
     ArgKind getArgumentKind() const {
```

Comment:
Are the changes in the file formatting only? If so, Can this be removed from this patch?

Reply:
Done.